cg/services/order_validation_service/rules/sample/utils.py

```diff
@@ -85,3 +85,10 @@ def is_invalid_well_format(sample: Sample) -> bool:
     if sample.is_on_plate:
         return not bool(re.match(correct_well_position_pattern, sample.well_position))
     return False
+
+
+def is_invalid_container_name(sample: Sample) -> bool:
```

Comment:
Hmmm thinking about it, it might make more sense to call this `is_container_name_missing` or something, since in theory the container name could be invalid in many ways.

Reply:
Yeah, that makes more sense. I will make a new commit.

Reply:
This is updated in the new commit.